easybuild/easyconfigs/g/GHC/ncurses-5.9_configure_darwin.patch

```diff
@@ -0,0 +1,13 @@
+# in relation to -no-cpp-precomp bug: https://lists.gnu.org/archive/html/bug-ncurses/2012-12/msg00001.html
+# tried/proposed by Fotis, meant for MacOSX compatible ncurses builds...
```

Comment:
@arielzn can you add the full author credentials here?

Reply:
ok, but i actually just pull it from ncurses tree, https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/ncurses/ncurses-5.9_configure_darwin.patch

i just put original pusher (Fotis) creds ?

Reply:
ah so `ncurses-5.9_configure_darwin.patch` is exactly the same as the one already in the repo?
then just remove it here, normally EB should be able to find it.

Reply:
it doesn't seem to be the case, all automatic tests failed badly when it was not there

see the boegelbot above

Reply:
hm, you're right. installing also fails unless you specifically add the path the the original patch.
you can ignore my stupid comments then and leave as is.